src/components/SmallBoard_m.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+import { useTranslation } from 'react-i18next';
+import { ReactNode } from "react";
```

Comment:
타입을 임포트할 때는 `type`을 적는 게 좋습니다. ([Ref.](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export))

> `import type` only imports declarations to be used for type annotations and declarations. It always gets fully erased, so there’s no remnant of it at runtime.

```suggestion
import type { ReactNode } from "react";
```